.github/workflows/nightly-fork.yml

```
@@ -0,0 +1,262 @@
+              - --filter=t3...
+              # scripts/update-release-package-versions.ts imports
+              # @effect/platform-node, which only @t3tools/scripts pulls in.
+              - --filter=@t3tools/scripts...
+              # The server serves the web client from dist/client.
+              - --filter=@t3tools/web...
+
+      - id: version
+        name: Resolve fork version
+        shell: bash
+        env:
+          RUN_STARTED_AT: ${{ github.run_started_at }}
```

actionlint (1.7.12)

[error] property "run_started_at" is not defined in object type {action: string; action_path: string; action_ref: string; action_repository: string; action_status: string; actor: string; actor_id: string; api_url: string; artifact_cache_size_limit: number; base_ref: string; env: string; event: object; event_name: string; event_path: string; graphql_url: string; head_ref: string; job: string; output: string; path: string; ref: string; ref_name: string; ref_protected: bool; ref_type: string; repository: string; repository_id: string; repository_owner: string; repository_owner_id: string; repository_visibility: string; repositoryurl: string; retention_days: number; run_attempt: string; run_id: string; run_number: string; secret_source: string; server_url: string; sha: string; state: string; step_summary: string; token: string; triggering_actor: string; workflow: string; workflow_ref: string; workflow_sha: string; workspace: string}

(expression)